doc/tutorials/imgproc/imgtrans/warp_affine/warp_affine.markdown

```diff
@@ -113,18 +113,18 @@ Explanation
     @code{.cpp}
     warp_dst = Mat::zeros( src.rows, src.cols, src.type() );
     @endcode
--#  **Affine Transform:** As we explained lines above, we need two sets of 3 points to derive the
-    affine transform relation. Take a look:
+-#  **Affine Transform:** As we explained in lines above, we need two sets of 3 points to derive the
+    affine transform relation. Have a look:
     @code{.cpp}
-    srcTri[0] = Point2f( 0,0 );
```

Comment:
There are no failed builds. Some of them are not scheduled, because there are documentation changes only - they will be checked before merge.